x/cdp/spec/README.md

```diff
@@ -0,0 +1,19 @@
+
+# `cdp`
+
+## Table of Contents
+
+1. **[Concepts](01_concepts.md)**
+2. **[State](02_state.md)**
+3. **[Messages](03_messages.md)**
+4. **[BeginBlock](04_begin_block.md)**
+5. **[Events](05_events.md)**
+6. **[Parameters](06_params.md)**
+
+## Overview
+
+The `x/cdp` module stores and manages Collateralized Debt Positions (or CDPs).
+
+A CDP enables the creation of an asset pegged to an external price (usually US Dollar) by collateralization with another asset. Collateral is locked in a CDP and new pegged asset can be minted up to approximately the value of the collateral. To unlock the collateral, the debt must be repaid by returning some pegged asset to the CDP at which point it will be burned and the collateral unlocked.
```

Comment:
```suggestion
A CDP enables the creation of a stable asset pegged to an external price (usually US Dollar) by collateralization with another asset. Collateral is locked in a CDP and new stable asset can be minted up to some fraction of the value of the collateral. To unlock the collateral, the debt must be repaid by returning some stable asset to the CDP at which point it will be burned and the collateral unlocked.
```